main.py

```diff
@@ -1,4 +1,9 @@
-
-def handleOddNumber(num): 
-    return (num * 3) + 1 
-
+
+def handleOddNumber(num): 
+    return (num * 3) + 1 
+
+def ImputIsEver(num):
```

Comment:
Nombre de la funcion no sigue el estilo de escritura camel case
```suggestion
def isEvenNumber(num):
```